CONTRIBUTING.md

```diff
@@ -0,0 +1,67 @@
+# iwant-bot
+---
+## Hello everyone!
+- :star2: If you are reading this document then I'm assuming that you are interested in contributing to the __iwant-bot__ project.
+- This project is made by Kiwi.com interns. 
+- All contributions are welcome: new use-cases, code, patches, bugfixes, issues, etc. 
+- This is just a set of guidelines, not rules. Use your best judgement, and feel free to propose changes in a pull request.
+
+
+## Ways to contribute
+---
+  - At first, check existing [issues] and our [pull requests] and try to find something 

```

Comment:
`You can check our existing issues and pull requests and try to find something that interests you`
also, fix links to issues and pull requests